app/tasks.py

```diff
@@ -93,6 +98,7 @@ def import_product_db(db: Session, batch_size: int = 1000) -> None:
             continue
         seen_codes.add(product_code)
 
+        product_lang = product.get("lang", product.get("lc", "en"))
```

Comment:
some products don't have the `lang` key

Reply:
Can you add it as a comment in the code?